test/fixtures/apps/check-status/app.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
-process.on('message', function(msg) {
-  if (msg.action === 'kill') process.exit(1);
-});
+const fs = require('fs');
+const path = require('path');
+
+module.exports = app => {
+  if (fs.existsSync(path.join(app.baseDir, 'logs/started'))) {
```

Comment:
why not notify app by message like below? why do use file ?

Reply:
Should exit after three times checking.